sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 1.4.0-beta.6 (2021-05-11)
+## 1.4.0 (2021-05-12)
 
 ### New Features
 
+- The Microsoft Authentication Library (MSAL) dependency versions have been updated to the latest
+    - Microsoft.Identity.Client version 4.30.1, Microsoft.Identity.Client.Extensions.Msal version 2.18.4.
```

Comment:
nit: I think this belongs under Fixes and Improvements rather than New Features.